apps/web/src/app/api/workspaces/[wsId]/upload/route.ts

```
@@ -13,7 +12,12 @@ export async function POST(req: Request, { params: { wsId } }: Props) {
   const url = new URL(req.url);
 
   // Extract query parameters
-  const filename = url.searchParams.get('filename') || '';
+  const fileName = url.searchParams.get('filename') || '';
+  const fileName = url.searchParams.get('filename') || '';
```

Biome

[error] Shouldn't redeclare 'fileName'. Consider to delete it or rename it.

'fileName' is defined here:


(lint/suspicious/noRedeclare)